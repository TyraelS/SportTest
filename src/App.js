import React from 'react';
import { ThemeProvider } from 'styled-components';
import * as themes from 'themes';

import AppStyle from './App.style';
import SideMenu from 'components/SideMenu';
import LeagueTabs from 'components/LeagueTabs';
import ThemesTab from 'components/ThemesTab';

function App({ theme }) {
  return (
    <ThemeProvider theme={themes[theme]}>
      <AppStyle>
        <SideMenu />
        <LeagueTabs />
        <ThemesTab />
      </AppStyle>
    </ThemeProvider>
  );
}

export default App;
