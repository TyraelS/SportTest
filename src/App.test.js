import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('Given the App component', () => {
  describe('when the component is rendered', () => {
    const component = shallow(<App theme={'light'} />);

    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
