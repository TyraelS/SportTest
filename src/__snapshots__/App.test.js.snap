// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the App component when the component is rendered should match the snapshot 1`] = `
<ThemeProvider
  theme={
    Object {
      "accent": "#bebebe",
      "leaguesItemBorder": "#000",
      "leaguesItemEven": "#fff",
      "leaguesItemOdd": "#eee",
      "regular": "#cecece",
      "textLeagues": "#000",
      "textSports": "#fff",
      "textThemes": "#000",
    }
  }
>
  <ForwardRef(styled.div)>
    <SideMenuContainer />
    <LeagueTabsContainer />
    <ThemesTabContainer />
  </ForwardRef(styled.div)>
</ThemeProvider>
`;
